glustercli/cmd/glustershd.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/gluster/glusterd2/pkg/glustershd/api"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpSelfHealCmd     = "Gluster Self Heal"
+	helpSelfHealInfoCmd = "Self Heal Info"
+)
+
+const (
+	infoSummary    = "info-summary"
```

Comment:
You can put these constants in the same const block. Even better if you get rid of them use it directly.